opm/simulators/linalg/FlowLinearSolverParameters.cpp

```diff
@@ -107,6 +122,11 @@ void FlowLinearSolverParameters::registerParameters()
          "cpr_trueimpes, cpr_trueimpesanalytic, amg or hybrid (experimental). "
          "Alternatively, you can request a configuration to be read from a "
          "JSON file by giving the filename here, ending with '.json.'");
+    Parameters::Register<Parameters::NlddLocalLinearSolver>
```

Comment:
This looks good, but I realized we have forgotten to change the help message for --linear-solver above when we changed the default linear solver, can you change it to say that cprw is default while changing this file anyway?